style.css

```diff
@@ -1 +1,215 @@
 /*  Add your styling here */
+body {
```

Comment:
Consider using stylu normalization - this is a good practice for any project. See normalize.css from the previous week.